Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/README.md

```diff
@@ -0,0 +1,38 @@
+# QBPSimulator
+Integration of the QBP process simulator into Apromore
+
+To build:
+
+    npm run build
+    node scripts/bundle-fix/index.js
+    cp dist/bpmn-modeler.development.js ../../Apromore-Frontend/src/bpmneditor/editor/bpmnio/bpmn-modeler.development.js
```

Comment:
As the module has been moved into a new folder, the path should be one more level:  "../../../Apromore-Frontend/src/bpmneditor/editor/bpmnio/bpmn-modeler.development.js"

Can we add this line for Windows because the path separator on Linux/Mac doesn't work: "copy dist\bpmn-modeler.development.js ..\..\..\Apromore-Frontend\src\bpmneditor\editor\bpmnio\bpmn-modeler.development.js"?

Reply:
Besides the build, can we add a line to run the test? Tried "npm run test" but it has some failures.

![Uploading image.png…]()